core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -143,6 +143,7 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
     }
     // Register web API for client
     RestAPIManager(hc).registerAll()
+    Thread.sleep(2000)
```

Comment:
I suggest to make it a configurable parameter (perhaps with a default value 2000)
This way you will be able to disable it if needed.